test/es6/default-splitscope-undodeferparse.js

```diff
@@ -0,0 +1,64 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function foo1(a, b) {
+    if (b != 10) {
+        print("FAILED")
+    } else {
+        print("PASSED");
```

Comment:
I don't we can have tests in RL like this because it only cares if the last line is 'PASSED'. If any previous tests fail, their failure will be ignored.

Can we use the unit test framework or use something other than 'PASSED' for the intermediate status?